bindata/manifests/machine-config/units/ovs-configuration.service.yaml

```diff
@@ -0,0 +1,9 @@
+name: ovs-configuration.service
```

Comment:
Why this file is needed?

Reply:
it's for my test environment. will remove it.